tests/foundry/test/UintTypeTest.t.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.13;
+
+contract UintTypeTest {
+    function setUp() public {}
+
+    function test_uint256(uint256 x) public pure {
+        assert(x == x);
+        assert(type(uint256).max >= x);
+    }
+
+    function testFail_uint256(uint256 x) public pure {
+        assert(type(uint256).max > x);
+    }
```

Comment:
Remember that there are 4 types of tests we should have:

- Tests that _should not_ fail and _do not_ fail.
- Tests that _should not_ fail and _do_ fail.
- Tests that _should_ fail and _do not_ fail.
- Tests that _should_ fail and _do_ fail.

Please make sure you have some tests of each type.

Reply:
Remember to add failing tests to the `tests/foundry/exclude` list.